Backpack/BottomSheet/Classes/BackpackFloatingPanelController.swift

```diff
@@ -0,0 +1,140 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FloatingPanel
+
+final class BackpackFloatingPanelController: FloatingPanelController {
```

Comment:
I think there's a difference here. In this case, I'm using `Backpack` to distinguish it from the class we're inheriting from: `FloatingPanelController`. It's "our" internal subclass.
The `BPK` prefix, is there to solve an Obj-C limitation which is lack of namespacing. So we would use `BPK` in order to avoid collision with other's frameworks classes or even app code from our users.

I would only add `BPK` prefix to Obj-C code, and to Obj-C names for our Swift classes, like this:
```swift
@objc(BPKBackpackFloatingPanelController)
public final class BackpackFloatingPanelController: FloatingPanelController {
    ...
}
```

I don't think we need it in this case, as this class is internal and not public.
Anyways, if you still think we should add the prefix to Swift names, I can do it. Let me know.

Reply:
Generally we've avoided including the name `Backpack` or the prefix `BPK` in Swift type names because it doesn't seem very "Swifty"

Reply:
I agree, adding a prefix in Swift makes no sense. That's why I didn't go for `BPK`, because I don't want it to be a prefix. I just wanted it to be visible that we are using our own subclass and not the regular `FloatingPanelController`.

I could also just remove `Backpack` and redefine `FloatingPanelController` within our module.

What do you think we should do?

Reply:
Dunno maybe we should keep it if we know there will be conflicts immediately. Or rename it something like `BottomSheetController`? Regardless the Objective-C name shouldn't include the `BPK` prefix. @georgegillams what do you think?

Reply:
I think avoiding the conflict would be best if there is another obvious name we can use instead, but if not let's just be explicit and call it BackpackFloatingPanelController